Tests/PhpUnit/ContainerHasParameterConstraintTest.php

```diff
@@ -29,23 +29,17 @@ public function containerBuilderProvider()
 
         $parameterName = 'parameter_name';
         $parameterValue = 'some value';
-        $containerWithParameter = $this->createMockContainerWithParameters(
-            array(
-                $parameterName => $parameterValue
-            )
-        );
-
         $wrongParameterValue = 'some other value';
 
         return array(
             // the container does not have the parameter
             array($emptyContainer, $parameterName, $parameterValue, true, false),
             // the container has the parameter but the values don't match
-            array($containerWithParameter, $parameterName, $wrongParameterValue, true, false),
+            array($this->createMockContainerWithParameters(array($parameterName => $parameterValue)), $parameterName, $wrongParameterValue, true, false),
```

Comment:
Looks reasonable to me as it doesn't make much sense to reuse the mocked object.